android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -115,6 +126,19 @@ public boolean onConsoleMessage(ConsoleMessage cm) {
         if (KMManager.isDebugMode()) {
           Log.d("KMEA", "Keyman JS Log: Line " + cm.lineNumber() + ", " + cm.sourceId() + ":" + cm.message());
         }
+
+        // Send console errors to Firebase Analytics.
+        // (Ignoring spurious message "No keyboard stubs exist = ...")
+        // TODO: Analyze if this error warrants reverting to default keyboard
```

Comment:
When that error exists, it should indicate that, at that point in time, there _is_ no default keyboard.  KMW is waiting on KMEA to provide that.

Reply:
KMEA used to provide KMW with a keyboard earlier in the pipeline; would it be possible for the app to provide that keyboard earlier so the message doesn't appear?  Our iOS app may still need that message.

Reply:
I'm not sure how it would be feasible for KMEA to provide a keyboard earlier.
By the time the keyboard.html page finishes loading, KMW has already flagged 
> No keyboard stubs exist - cannot initialize keyboard!

Reply:
> By the time the keyboard.html page finishes loading

Why wait?  As long as the script has loaded, keyboards can be submitted to KMW.

Reply:
This goes back to #456 when I was reorganizing the default keyboard assets in preparation for the KMP distribution feature. 

You added the error message (b3b3922fb62eba14b3436e8c5147687fdefd66bc) to troubleshoot embedded keyboard registration. Maybe it was to confirm the namespacing?

Reply:
Per discussion with @mcdurdin, I'm going to merge this PR and continue the discussion on KMEA keyboard registration on issue #664